test-fixtures/update.py

```diff
@@ -21,7 +21,7 @@
 ]
 '''
 pattern = 'osx-xcode-'
-BITRISE_YML = '../bitrise.yml'
+BITRISE_YML = 'bitrise.yml'
```

Comment:
Not sure if it would be better to add the `print(os.getcwd())` statement before this line...

Reply:
Added both for now 👍🏼